charts/kuberpult/Chart.yaml

```diff
@@ -1,29 +0,0 @@
-apiVersion: v2
```

Comment:
Why removing this???

Reply:
In all helm charts that we use `Chart.yaml` and `values.yaml` are defined there

Reply:
yes, but they are generated automatically, so we don't need to check them in.